.travis.yml

```diff
@@ -12,36 +12,22 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=latest
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
```

Comment:
These tests cannot be removed. Please add again. Thanks!

Reply:
Should they be added to each version block? That was unclear to me given they were only present on the 7.1 version previously.

Reply:
Sorry, only for one version. You can use version 7.4.

You can compare with other components. For example: https://github.com/laminas/laminas-hydrator/blob/4.1.x/.travis.yml